openapi/paths/journal-accounts@{id}.yaml

```
@@ -0,0 +1,64 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Journal
+  summary: Retrieve journal account
+  operationId: GetJournalAccount
+  x-sdk-operation-name: getAccount
+  description: Retrieves a journal account with a predefined identifier.
```

nitpick: based on our style
```suggestion
  description: Retrieves a journal account with a specified ID.
```